keras/backend/mxnet_backend.py

```diff
@@ -4230,6 +4216,25 @@ def predict_function(inputs):
 
             self.predict_function = predict_function
 
+        def _get_mxnet_context(self, context):
+            mxnet_context = []
+            if context is None:
+                mxnet_context.append(mx.current_context())
+            elif isinstance(context, str):
+                mxnet_context.append(context)
+            else:
```

Comment:
I think it is better to assert context is a list and raise exception if not one of the cases.

Reply:
It is more native for users to say model.compile(....., context="cpu") than model.compile(...., context="[cpu]".
Hence, handling it for users without asserting.

Reply:
got it thank you !